src/einsteinpy/symbolic/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .scalar_factor import scalar_factor, scalar_factor_derivative
```

Comment:
By removing all the imports, I mean make the init file empty.

Reply:
Also, remove the irrelevant imports from init of utils :) 

And there is a try except block in the christoffel.py. Remove that as well and directly import sympy.

Reply:
Done